### ISSUE TITLE
UWP support is languishing.  Shameful.

### ISSUE THREAD

**Issue by Noemata** [microsoft/dotnet#1171]
UWP support is languishing.  It's shameful this time around.  As cool as it is to see a flurry of activity around WPF all of a sudden, that too is in some half baked state at present.  Please just finish one thing all the way before moving on to the next.

What's going on?  This is starting to smell like the Silverlight death.

Is the Office Team that hung up on Win32 and all the old plodding code that comes with it?

**Comment by Gavin-Williams** [comment]
Yeah, it's bad. There are many great things about UWP. But there's a lot of confusion in Microsoft about what it is, what it's future is, How it's going to be integrated into .Net 5. The recent .Net 5 preview article didn't mention UWP once. That show's that Microsoft don't even know what they've got. Some do, but there are heavy weights that are fixated on their own domains.

The C# team ignore game development completely - which is just naive. They need to put some game developers onto the C# team to open up their eyes. It's incredible how blind they are.
